src/test/java/org/broadinstitute/hellbender/engine/dataflow/ReadsSourceTest.java

```diff
@@ -21,6 +23,7 @@
 public final class ReadsSourceTest extends BaseTest {
 
     private final File bam = new File(this.getToolTestDataDir(), "count_reads_sorted.bam");
+    final String hiSeqBam = "src/test/resources/org/broadinstitute/hellbender/tools/BQSR/HiSeq.1mb.1RG.2k_lines.bam";
```

Comment:
Could you copy this file to the `src/test/resources/org/broadinstitute/hellbender/engine/dataflow/ReadsSource/`.  It should be identified as a duplicate by git so it won't increase repo size, but it will save confusion if someone edits it in the future.